randovania/games/fusion/logic_database/Sector 1 SRX.txt

```diff
@@ -540,16 +540,26 @@ Extra - room_id: [16]
       All of the following:
           Screw Attack
           Any of the following:
-              # Get up
               Space Jump
               All of the following:
-                  # Jump on frozen Space Pirates TODO: someone please get a video of this. Also, how much harder is this with Diffusion, worth splitting out from template?
-                  Can Freeze Enemies
                   Any of the following:
-                      # HJ method. Some RNG involved due to pirates moving.
-                      High Jump and Stand On Frozen Enemies (Advanced)
-                      # Walljump method. Heavy RNG involved due to pirates jumping
-                      Stand On Frozen Enemies (Hypermode) and Wall Jump (Beginner)
+                      # Wall Jump reqs
+                      All of the following:
+                          # Single Wall Jump
+                          Can Single Walljump
+                          Any of the following:
+                              Wall Jump (Advanced)
+                              High Jump and Wall Jump (Intermediate)
+                      # Or just stand on all the pirates I guess TODO: get me a vid of this NHJ or else I'm not putting that in here
+                      High Jump and Stand On Frozen Enemies (Hypermode) and Wall Jump (Beginner)
+                  Any of the following:
+                      # Ice req
+                      Any of the following:
+                          # https://youtu.be/6ha7hZ0iW1I
+                          Ice Beam
+                          Ice Missile Data and Missiles
+                      # https://youtu.be/o91JwaZoFTg
+                      Diffusion Missile Data and Missiles and Stand On Frozen Enemies (Hypermode)
```

Comment:
Doing this with Diffusion depends on whether you do the SWJ strat or the stand-on-pirates strat. Probably should have 5 Missiles just in case:tm: